modules/ipi-verifying-nodes-after-installation.adoc

```diff
@@ -0,0 +1,64 @@
+//
+// *
+
+:_module-type: PROCEDURE
+[id="ipi-verifying-nodes-after-installation_{context}"]
+= Verifying node state after installation
+
+[role="_abstract"]
+//Short introductory paragraph that provides an overview of the module. The text that immediately follows the abstract tag is used for search metadata.
+The {product-title} installation completes when the following installation health checks are successful:
+
+* The provisioning host can access the OpenShift web console.
+
+* All master nodes are ready.
+
+* All cluster Operators are available.
+
+[NOTE]
+====
+It can take some time after cluster installation is successfully completed before all worker nodes report as `READY`. The Cluster Operator continuously attempts to provision worker nodes for a set period of time. The state of a worker node only changes to `DEGRADED` when the Cluster Operator fails to provision the worker node within this time period. You can define this time period in the `install-config.yaml` file.
+==== 
+
+After your installation completes, you can continue to monitor the condition of the nodes in your cluster using the following steps. 
+
+.Prerequisites
+//A bulleted list of conditions that must be satisfied before the user starts following this module.
+* The installation program resolves successfully in the terminal.
+
+.Procedure
+//Start each step with an active verb. Use an unnumbered bullet (*) if the procedure includes only one step.
+. Show the status of all worker nodes:
++
+[source,terminal]
+----
+oc get nodes
+----
++
+.Example output
+[source,terminal]
+NAME                           STATUS   ROLES    AGE   VERSION
+example-compute1.example.com   Ready    worker   13m   v1.21.6+bb8d50a
+example-compute2.example.com   Ready    worker   13m   v1.21.6+bb8d50a
+example-compute4.example.com   Ready    worker   14m   v1.21.6+bb8d50a
+example-control1.example.com   Ready    master   52m   v1.21.6+bb8d50a
+example-control2.example.com   Ready    master   55m   v1.21.6+bb8d50a
+example-control3.example.com   Ready    master   55m   v1.21.6+bb8d50a
+
+. Show the phase of all worker machine nodes:
++
+----
+oc get machines -A
+----
++
+.Example output
+[source,terminal]
+NAMESPACE               NAME                           PHASE         TYPE   REGION   ZONE   AGE
+openshift-machine-api   example-zbbt6-master-0         Running                              95m
+openshift-machine-api   example-zbbt6-master-1         Running                              95m
+openshift-machine-api   example-zbbt6-master-2         Running                              95m
+openshift-machine-api   example-zbbt6-worker-0-25bhp   Running                              49m
+openshift-machine-api   example-zbbt6-worker-0-8b4c2   Running                              49m
+openshift-machine-api   example-zbbt6-worker-0-jkbqt   Provisioned                          49m
+openshift-machine-api   example-zbbt6-worker-0-qrl5b   Running                              49m
```

Comment:
Maybe we should use here a "successful" example (that is, all amchines _Running_).

Reply:
Replaced example with all "running" machines